parentheses.py

```diff
@@ -0,0 +1,53 @@
+import math
+from operator import add, sub, mul
+
+
+operators={"+": add, "-": sub, "*": mul}
+
+
+def calculate(first_operand, second_operand, operator) -> int:
+    """
+    Return the value after evaluating the expression
+    """
+    return operators[operator](first_operand, second_operand)
+
+
+def MinandMax(maximum_numbers, minimum_numbers, i, j, operator) -> int:

```

Comment:
```suggestion
def min_and_max(minimum_numbers: float, maximum_numbers: float, i: int, j: int, operator: str) -> int:

```

Reply:
If the function is called `min_and_max()` then the order of the parameters can not be the opposite.